scenes/player_test/player_test.gd

```diff
@@ -0,0 +1,25 @@
+extends CharacterBody2D
```

Comment:
Test file. If not longer needed consider removing it.

Reply:
Is used to demostrate how to use StateMachine